js/data_v4.js

```diff
@@ -298,6 +302,14 @@ function populateThisMonth(pingList) {
         if (!isPastNDays(timestampCreated, 35, now)) {
             continue;
         }
+
+        // We only parse main and crash-ping(s) in PingAccumulator,
+        // so don't load other ping types contents. Do it here so we
+        // still see the link in the raw data section.
```

Comment:
"Do it here"?
How about just "The other pings are still visible in the raw data section."